README.md

```diff
@@ -6,9 +6,20 @@
    pip install poetry && poetry install && poetry run pre-commit install
 ```
 
-## Testing
+## Testing / quality checks
 
-The short tests that run against the `terraform plan` for a module can be run by `cd`-ing into the module folder and running:
+Various quality checks are run in AWS Codebuild in the `platform-tools` account for any push to a pull request branch:
+
+* [Checkov](https://www.checkov.io/)
+* [terraform fmt](https://developer.hashicorp.com/terraform/cli/commands/fmt)
+* [terraform validate](https://developer.hashicorp.com/terraform/cli/commands/validate)
+* [tflint](https://github.com/terraform-linters/tflint])
+* [terraform test](https://developer.hashicorp.com/terraform/cli/commands/test) - plan style
+* Todo: [terraform test](https://developer.hashicorp.com/terraform/cli/commands/test) - end to end style
```

Comment:
not entirely sure about these two lines atm, what's `plan style` and `end to end style`? just wondering about an explanation, other than that the PR is 👍🏻